.ci/set_tag_version_images.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright (c) 2017-2021 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
```

Comment:
paul

Reply:
are you saying "this should have EPL v2 on it?"